src/transformers/models/olmo_1124/modular_olmo_1124.py

```diff
@@ -474,3 +475,9 @@ class Olmo1124ForCausalLM(OlmoForCausalLM):
     def __init__(self, config: Olmo1124Config):
         super().__init__(config)
         self.model = Olmo1124Model(config)
+
+
+class Olmo1124ForSequenceClassification(OlmoForSequenceClassification):
+    def __init__(self, config: Olmo1124Config):
+        super().__init__(config)
+        self.model = Olmo1124Model(config)
```

Comment:
I added those here and ran `python utils/modular_model_converter.py --files_to_parse src/transformers/models/olmo_1124/modular_olmo_1124.py`, but nothing seems to be generated